rest-services/server/src/main/java/com/_4point/aem/docservices/rest_services/server/assembler/AssembleDocuments.java

```diff
@@ -0,0 +1,209 @@
+import com._4point.aem.docservices.rest_services.server.Exceptions.InternalServerErrorException;
+import com._4point.aem.docservices.rest_services.server.Exceptions.NotAcceptableException;
+import com._4point.aem.docservices.rest_services.server.ServletUtils;
+import com._4point.aem.fluentforms.api.Document;
+import com._4point.aem.fluentforms.api.DocumentFactory;
+import com._4point.aem.fluentforms.api.assembler.AssemblerOptionsSpec;
+import com._4point.aem.fluentforms.api.assembler.AssemblerResult;
+import com._4point.aem.fluentforms.api.assembler.AssemblerService;
+import com._4point.aem.fluentforms.api.assembler.AssemblerService.AssemblerServiceException;
+import com._4point.aem.fluentforms.impl.UsageContext;
+import com._4point.aem.fluentforms.impl.assembler.AdobeDocAssemblerServiceAdapter;
+import com._4point.aem.fluentforms.impl.assembler.AssemblerOptionsSpecImpl;
+import com._4point.aem.fluentforms.impl.assembler.AssemblerServiceImpl;
+import com._4point.aem.fluentforms.impl.assembler.TraditionalDocAssemblerService;
+
+@SuppressWarnings("serial")
+@Component(service = Servlet.class, property = {
+		Constants.SERVICE_DESCRIPTION + "=FormsService.AssembleDocuments Service",
+		"sling.servlet.methods=" + HttpConstants.METHOD_POST })
+@SlingServletPaths("/services/AssemblerService/AssembleDocuments")
+public class AssembleDocuments extends SlingAllMethodsServlet {
+
+	private static final Logger log = LoggerFactory.getLogger(AssembleDocuments.class);
+
+	@Reference
+	private com.adobe.fd.assembler.service.AssemblerService adobeAssembleService;
+
+	private final Supplier<TraditionalDocAssemblerService> assemblerServiceFactory = this::getAdobeAssemblerService;
+	
+
+	private final DocumentFactory docFactory = DocumentFactory.getDefault();
+
+	@Override
+	protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response)
+			throws ServletException, IOException {
+		try {
+			this.processInput(request, response);
+		} catch (BadRequestException br) {
+			log.warn("Bad Request from the user", br);
+			response.sendError(SlingHttpServletResponse.SC_BAD_REQUEST, br.getMessage());
+		} catch (InternalServerErrorException ise) {
+			log.error("Internal server error", ise);
+			response.sendError(SlingHttpServletResponse.SC_INTERNAL_SERVER_ERROR, ise.getMessage());
+		} catch (NotAcceptableException nae) {
+			log.error("NotAcceptable error", nae);
+			response.sendError(SlingHttpServletResponse.SC_NOT_ACCEPTABLE, nae.getMessage());
+		} catch (Exception e) { // Some exception we haven't anticipated.
+			log.error(e.getMessage() != null ? e.getMessage() : e.getClass().getName(), e); // Make sure this gets into
+																							// our log.
+			throw e;
+		}
+	}
+
+	private void processInput(SlingHttpServletRequest request, SlingHttpServletResponse response)
+			throws BadRequestException, InternalServerErrorException, NotAcceptableException {
+		// TODO Auto-generated method stub
```

Comment:
Looks like some of the TODO comments were left in from stub generation.  These should be removed.